.github/insert_printable_hyperlinks.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import fileinput
+import pathlib
+import re
+
+HYPERLINKS_TO_REPLACE = [
+    "AIrules",
+    "Ability",
+    "Combat",
+    "Defend",
+    "Difficulty",
+    "End",
+    "Endcombat",
+    "Level",
+    "Movement",
+    "Playerdecks",
+    "Quick",
+    "Trading",
+    "Walls",
+]
+SECTIONS_DIRECTORY = "sections"
+
+
+def _list_tex_files(directory):
+    return pathlib.Path(f"./{directory}").glob("*.tex")
+
+
+def _replace_line(line):
+    for pattern in HYPERLINKS_TO_REPLACE:
+        match = re.search(f"hyperlink{{{pattern}", line)
+        line = (
+            line.replace(f"hyperlink{{{pattern}", f"pagelink{{{pattern}")
+            if match
+            else line
+        )
+    return line
```

Comment:
Add typehint and make it simpler
```suggestion
def _replace_line(line: str):
    for pattern in HYPERLINKS_TO_REPLACE:
        if re.search(f"hyperlink{{{pattern}", line):
            line = line.replace(f"hyperlink{{{pattern}", f"pagelink{{{pattern}")
    return line
```